src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag

```diff
@@ -13,18 +13,18 @@
   </h3>
   <br>
   <div class="form-group row">
-    <div class="col-xs-6 col-sm-5 col-md-3 cursor-pointer"
-        title="<%= Const.Tooltips.STUDENT_PROFILE_PICTURE %>"
-        data-toggle="tooltip"
-        data-placement="top">
+    <div class="col-xs-6 col-sm-5 col-md-3 cursor-pointer">
       <img id="profilePic"
           src="${profile.pictureUrl}"
           class="profile-pic"
           data-toggle="modal"
           data-target="#studentPhotoUploader"
           data-edit="${profile.editingPhoto}">
     </div>
-    <div class="">
+    <div class="col-xs-6 col-sm-5 col-md-3 cursor-pointer"
```

Comment:
I think the `cursor-pointer` here is unnecessary.

Reply:
Ok, thaks

Reply:
@sukanta-27 Done

Reply:
@sukanta-27 Done, but the latest build was failed. Something went wrong with - "nvm install node" failed and exited with 3 during.